src/components/FileAnnotation/FileEditor.js

```diff
@@ -18,13 +19,22 @@ const FileEditor = ({
   onNameChange,
   onDescriptionChange,
   history,
+  showStatus,
+  renderButtons,
 }) => {
-  const [editing, setEditing] = useState(false);
-
+  const [editingName, setEditingName] = useState(!showStatus);
+  let editButtonClass = classes('pl-20', {hidden: !showStatus});
+  let badgeClass = classes(showStatus ? 'flex' : 'hidden', 'row-2', 'cell-12');
   return (
-    <form onSubmit={e => onSubmit(e)} className="FileEditor">
+    <form
+      onSubmit={e => {
+        e.preventDefault();
+        onSubmit(e);
+      }}
+      className="FileEditor"
+    >
       <GridContainer collapsed="rows">
-        {editing ? (
+        {editingName ? (
           <label className="row-1 cell-12 flex justify-between">
```

Comment:
add a `HtmlFor` prop, this will render the `for` prop that points to the input `name` for this label  so 
```
<label for="file_name_input">...</label>
<input name="file_name_input" />
```